scripts/airgapped/save-images-to-tar.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import logging
+import os
+import subprocess
+
+import docker
+
+from utils import (delete_file_if_exists, get_images_list_from_file,
+                   get_or_pull_image)
+
+cli = docker.client.from_env()
+
+log = logging.getLogger(__name__)
+
+
+def save_image(image_nm) -> str:
+    """Given an Image object, save it as tar."""
+    get_or_pull_image(image_nm)
+    file_name = "%s.tar" % image_nm
+    file_name = file_name.replace("/", "-").replace(":", "-")
+    if os.path.isfile(file_name):
+        log.info("Tar '%s' already exists. Skipping...", file_name)
+        return file_name
+
+    log.info("%s: Saving image to tar '%s'.", image_nm, file_name)
+    for i in range(10):
+        # We've seen that sometimes we get socket timeouts. Try 10 times
+        try:
+            with open(file_name, "w+b") as f:
+                subprocess.run(["docker", "save", image_nm], stdout=f)
```

Comment:
Would skopeo be of better help?
An example from a different context:
```
skopeo --insecure-policy copy oci-archive:alertmanager_0.24.0_amd64.rock docker-daemon:alertmanager:0.24.0
```

Reply:
I'll go on and resolve this conversation, but feel free to re-open if you feel strongly about this @sed-i :)